src/components/media/audio/sound.tsx

```diff
@@ -0,0 +1,232 @@
+        />
+      )
+    }
+  }
+
+  function renderMobileView() {
+    if (audioRef.current) {
+      return (
+        <>
+          <Box className="rustic-sound-controls">
+            <TimeIndicator
+              elapsedTimeInSeconds={elapsedTime}
+              durationTimeInSeconds={audioRef.current.duration}
+              style="wide"
+            />
+            <ProgressSlider mediaElement={audioRef.current} />
+            {renderTitle()}
+
+            <Box className="rustic-sound-bottom-controls">
+              <Box className="rustic-sound-bottom-controls-left">
+                {renderTimeControls()}
+              </Box>
+
+              <VolumeSettings mediaElement={audioRef.current} />
+
+              <Box className="rustic-sound-bottom-controls-right">
+                {renderTranscriptToggle()}
+
+                <Box>
+                  {renderCaptionsToggle()}
+                  <PlaybackRateButton mediaElement={audioRef.current} />
+                </Box>
+              </Box>
+            </Box>
+
+            {renderTranscript()}
+          </Box>
+        </>
+      )
+    }
+  }
+
+  function renderDesktopView() {
+    if (audioRef.current) {
+      return (
+        <>
+          {renderTitle()}
+
+          <Box className="rustic-sound-controls">
+            <ProgressSlider mediaElement={audioRef.current} />
+
+            <Box className="rustic-sound-bottom-controls">
+              <Box className="rustic-sound-bottom-controls-left">
+                {renderTimeControls()}
+                {renderCaptionsToggle()}
+
+                <PlaybackRateButton mediaElement={audioRef.current} />
+                <VolumeSettings mediaElement={audioRef.current} />
+                <TimeIndicator
+                  elapsedTimeInSeconds={elapsedTime}
+                  durationTimeInSeconds={audioRef.current.duration}
+                  style="condensed"
+                />
+              </Box>
+
+              <Box className="rustic-sound-bottom-controls-right">
+                {renderTranscriptToggle()}
+              </Box>
+            </Box>
+
+            {renderTranscript()}
+          </Box>
+        </>
+      )
+    }
+  }
+
+  const renderCurrentView = isMobile ? renderMobileView : renderDesktopView
+
+  if (errorMessage.length > 0) {
+    return (
+      <Alert severity="error" data-cy="error">
+        {errorMessage}
+      </Alert>
+    )
+  }
+
+  return (
+    <Box className="rustic-sound" data-cy="audio">
+      <video
+        src={props.src}
+        data-cy="audio-element"
+        ref={audioRef}
+        className={areCaptionsShown ? 'rustic-has-captions' : ''}
+      >
+        {areCaptionsShown && (
+          <track src={props.captions} kind="captions" default />
+        )}
+      </video>
+
+      {isLoading || !audioRef.current ? (
+        <CircularProgress data-cy="spinner" />
+      ) : (
+        <>{renderCurrentView()}</>
+      )}
```

Comment:
Ternary operations are for simple actions. Could you use if/else statement instead so it's more readable.

Reply:
`!audioRef.current` is redundant, so I've removed it if it makes it more readable.

Reply:
agree with @lyjeileen . Please update this